book/controller.rst

```diff
@@ -211,13 +218,13 @@ simply creating a controller method and an associated route.
 
 Simple, right?
 
-.. sidebar:: The AppBundle:Hello:index controller syntax
-
-    If you use the YML or XML formats, you'll refer to the controller using
-    a special shortcut syntax: ``AppBundle:Hello:index``. For more details
-    on the controller format, see :ref:`controller-string-syntax`.
+.. sidebar:: Logical controller name
```

Comment:
I'm reversing this - I think the user will recognize what we mean by `AppBundle:Hello:index` - but they may not know what a "logical name" is at this point :)